app/javascript/courses/report/components/CoursesReport__Overview.re

```diff
@@ -60,22 +60,30 @@ let doughnutChart = (color, percentage) => {
     </text>
   </svg>;
 };
-let targetsCompletionStatus = (targetsCompleted, totalTargets) => {
+let targetsCompletionStatus = overview => {
+  let targetsCompleted = overview |> StudentOverview.targetsCompleted;
+  let totalTargets = overview |> StudentOverview.totalTargets;
+  let targetsPendingReview = overview |> StudentOverview.targetsPendingReview;
+  let targetsNotAttempted =
```

Comment:
Let's keep the variable name similar to the status which is `incompleteTargets`

Reply:
Actually, I prefer this naming. This makes it clear what the status is, whereas the word `incomplete` could mean either _not attempted, pending review, or failed_. 🤔

Reply:
Little unsure about the condition where a student fails in a submission, and we still say not attempted.

Reply:
@mahesh-krishnakumar Talked to Bodhi just now. You're right. _Incomplete_ works better.